vault/diagnose/mock_storage_backend.go

```diff
@@ -0,0 +1,73 @@
+package diagnose
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/sdk/physical"
+)
+
+const timeoutCallRead string = "lag Read"
```

Comment:
could you do a `const` block here?

e.g.
```
const (
    thing1 = "hello"
    thing2 = "world"
   ...
)
```

Reply:
i'm only gonna put one comment for this class of thing, but could you also do the same with other consts and vars? i think it makes it a bit cleaner

Reply:
Yes, will do!